dialog/lib/dialog.ts

```diff
@@ -457,6 +457,10 @@ export class Dialog extends LitElement {
   private handleDialogDismiss(e: Event) {
     if (e.type === 'cancel') {
       this.currentAction = this.escapeKeyAction;
+      if (this.currentAction === '') {
+        e.preventDefault();
```

Comment:
Yes because `handleDialogDismiss` is called from `@cancel` event handler from `dialog` element which is  triggered when escape key is pressed,  and the only way to prevent this `<dialog>` default closing behavior is actually calling `preventDefault`.

Reply:
Thanks!